src/collection-view.js

```diff
@@ -28,6 +28,7 @@ Marionette.CollectionView = Marionette.View.extend({
 
     Marionette.View.apply(this, arguments);
 
+    this.on('before:show', this._onBeforeShowCalled);
```

Comment:
Ask @samccone?  I think `this.on('show'...)` came from #d1afdcd6e7aa33c6234912d4684760915bc32178.